distdir_deps.bzl

```diff
@@ -227,10 +227,10 @@ DIST_DEPS = {
     "remote_java_tools_linux": {
         "aliases": ["remote_java_tools_test_linux", "remote_java_tools_linux_for_testing"],
         "archive": "java_tools_linux-v11.0.zip",
```

Comment:
Do these lines need to be updated?

Reply:
They sure do. Thanks!

Reply:
Ah.. sorry for missing these!

Reply:
Fixes here https://github.com/bazelbuild/bazel/pull/12824